auto_object.ts

```diff
@@ -106,10 +106,9 @@ function getAutoPlacements(stat: any, atype: string) {
       let shape: string = area.Shape;
       let num: number = fieldArea.getCurrentAreaNum(x, z);
 
-      parts = parts.sort((a: any, b: any) => b.num - a.num);
-      parts = parts.map((item: any) => `${real_name(item.name)}: ${item.num} %`);
+      parts = parts.map((item: any) => `- ${real_name(item.name)}: ${item.num}`);
 
-      let key = `${atype} RNG<br/>${parts.join('<br/>')}<br/>Field Map Area ${num}<br/>#${n}`;
+      let key = `${atype}<br/>${parts.join('<br/>')}<br/>Field Map Area: ${num}<span style="display:none">${n}</span>`;
```

Comment:
Hmm, it's a bit weird to have hidden HTML in the key... Actually I think there shouldn't be any HTML at all. Formatting should be done on the client side. Can we turn the entry format into the following?

```js
[
  {
    "type": atype,
    "field_map_area": num,
    "geom": geom,
    ... // any other useful properties
]
```

Basically just dump an array of objects with any necessary information and let the client do whatever it wishes with the data.